eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/RatifySpec.hs

```diff
@@ -624,6 +627,117 @@ votingSpec =
+          it "Directly" $ do
+            -- Only modify the applicable thresholds
+            modifyPParams $ \pp ->
+              pp
+                & ppDRepVotingThresholdsL
+                  .~ def
+                    { dvtCommitteeNormal = 51 %! 100
+                    , dvtCommitteeNoConfidence = 51 %! 100
+                    }
+                & ppCommitteeMaxTermLengthL .~ EpochInterval 20
+                & ppGovActionDepositL .~ Coin 600_000
+            -- Setup DRep delegation without stake #1
+            (drepKH1, stakingKH1) <- setupDRepWithoutStake
+            -- Setup DRep delegation #2
+            (_drepKH2, _stakingKH2, _paymentKP2) <- setupSingleDRep 1_000_000
+            -- Make a note of the reward account for the delegator to DRep #1
+            dRepRewardAccount <- getRewardAccountFor $ KeyHashObj stakingKH1
+            -- Submit the first committee proposal, the one we will test active voting stake against.
+            -- The proposal deposit comes from the root UTxO
+            cc <- KeyHashObj <$> freshKeyHash
+            let addCCAction = UpdateCommittee SNothing mempty (Map.singleton cc 10) (75 %! 100)
+            addCCGaid <-
+              submitProposal $
+                ProposalProcedure
+                  { pProcDeposit = Coin 600_000
+                  , pProcReturnAddr = dRepRewardAccount
+                  , pProcGovAction = addCCAction
+                  , pProcAnchor = def
+                  }
+            -- Submit the vote from DRep #1
+            submitVote_ VoteYes (DRepVoter $ KeyHashObj drepKH1) addCCGaid
+            passNEpochs 2
+            -- The vote should not result in a ratification
+            isDRepAccepted addCCGaid `shouldReturn` False
+            getLastEnactedCommittee `shouldReturn` SNothing
+            -- Submit another proposal to bump up the active voting stake
+            anotherCC <- KeyHashObj <$> freshKeyHash
+            let anotherAddCCAction = UpdateCommittee SNothing mempty (Map.singleton anotherCC 10) (75 %! 100)
+            _anotherAddCCGaid <-
+              submitProposal $
+                ProposalProcedure
+                  { pProcDeposit = Coin 600_000
+                  , pProcReturnAddr = dRepRewardAccount
+                  , pProcGovAction = anotherAddCCAction
+                  , pProcAnchor = def
+                  }
+            passNEpochs 2
+            -- The same vote should now successfully ratify the proposal
+            getLastEnactedCommittee `shouldReturn` SJust (GovPurposeId addCCGaid)
+          it "After switching delegations" $ do
+            -- Only modify the applicable thresholds
+            modifyPParams $ \pp ->
+              pp
+                & ppDRepVotingThresholdsL
+                  .~ def
+                    { dvtCommitteeNormal = 51 %! 100
+                    , dvtCommitteeNoConfidence = 51 %! 100
+                    }
+                & ppCommitteeMaxTermLengthL .~ EpochInterval 20
```

Comment:
Same comment about unnecessary setting of `ppCommitteeMaxTermLengthL` (and probably also of `dvtCommitteeNoConfidence`)